docs/jupyterlite.rst

```diff
@@ -0,0 +1,25 @@
+Try out now with JupyterLite_
+-----------------------------
+
+.. admonition:: To try pyhf code in the browser with the Pyolite JupyterLite kernel:
```

Comment:
What do people think? If what is currently there is too long and technical we can shorten it to something like

```suggestion
.. admonition:: To try pyhf code in the browser:
```